mypy/test/teststubtest.py

```diff
@@ -593,6 +593,51 @@ class BadReadOnly:
             """,
             error="BadReadOnly.f",
         )
+        yield Case(
```

Comment:
I think this test case duplicates `Good`, so we should remove it (feel free to rename `Good.f` to `Good.read_only_attr`)

Reply:
Thanks — done!